Plugins/NativeServerPlugin/StreamingNativeServerPlugin.vcxproj

```diff
@@ -91,7 +91,7 @@
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>_WINDOWS;_CRT_SECURE_NO_WARNINGS;NOMINMAX;WEBRTC_WIN;TOOLKIT3DLIBRARY_EXPORTS;_DEBUG;WIN32;%(PreprocessorDefinitions);</PreprocessorDefinitions>
       <SDLCheck>true</SDLCheck>
-      <AdditionalIncludeDirectories>inc;$(ProjectDir)..\..\Libraries\WebRTC\headers;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>inc;$(ProjectDir)..\..\Libraries\WebRTC\headers;$(ProjectDir)..\..\Libraries\Freeglut\include\GL;$(ProjectDir)..\..\Libraries\glext\include\GL;$(ProjectDir)..\..\Libraries\Glew\include\GL;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>

```

Comment:
these new values should go in `exports.props` as well, that's what drives include directories for downlevel projects. 

this should remove the need to add these new includes explicitly to `SpinningCubeServerGL.vcxproj` so they should then be removed from there.

Reply:
great point! I will move the includes to exports.props